components/benefit_sponsors/app/models/benefit_sponsors/serializers/csv/benefit_application_issuer.rb

```diff
@@ -0,0 +1,22 @@
+module BenefitSponsors
+  module Serializers
+    class BenefitApplicationIssuer
+      # employer_hbx_id,employer_feinlan_year_startlan_year_end,carrier_hbx_id,carrier_fein
+      def self.to_csv(benefit_application)
+        benefit_application.benefit_packages.inject([]) do |list, benefit_package|
+          benefit_package.sponsored_benefits.each do |sponsored_benefit|
+            sponsored_benefit.products(Date.today + 1.month).each do |product|
```

Comment:
If we do this on a currently active plan year that is scheduled to renew next month, the list will be empty - which is not what we want.

In that case we want both the current and future plan year, and the associated products.

Make sure you aren't looking for packages out past the end of the benefit application period.

This should probably have been caught by the specs.

Reply:
I changed this use the `benefit_application.start_on` which will ensure we get the right products because they should cover when application starts.